.dagger/build/builder.go

```diff
@@ -436,17 +436,16 @@ func (build *Builder) cniPlugins() []*dagger.File {
 func (build *Builder) dumbInit() *dagger.File {
 	// dumb init is static, so we can use it on any base image
 	return dag.
-		Alpine(dagger.AlpineOpts{
-			Branch: consts.AlpineVersion,
-			Packages: []string{
-				"bash",
-				"build-base",
-			},
-			Arch: build.platformSpec.Architecture,
-		}).
 		Container().
+		From(consts.AlpineImage).
```

Comment:
Annoyingly, we need to go back to relying on this instead of using our `alpine` module, since the database isn't in a valid state after we finish with it.

Working on that here: https://github.com/dagger/dagger/pull/9117